pkg/users/index.html

```diff
@@ -433,7 +433,7 @@ <h4 class="modal-title" translatable="yes">Add public key</h4>
       <div class="pf-c-alert__icon">
           <i class="fa fa-info-circle" aria-hidden="true"></i>
       </div>
-      <h4 translatable class="pf-c-alert__title">The user must log out and log back in to fully change roles.</h4>
```

Comment:
This seems like it never worked and this string was missing in translations. (And weirdly we had `translatable` as string to be translated instead)

Reply:
Some HTML attributes duplicate themselves if they're included without a value. For example: `disabled` becomes `disabled="disabled"`. (Perhaps something like that happened here?)